src/components/orderTaking/orderScreen.re

```diff
@@ -28,20 +39,39 @@ type action =
   | DeselectTag
   | LoadOrder(Order.orderVm)
   | CloseOrderScreen
-  | RemoveOrderItem(OrderItem.t)
   | ChangePaidDate(float)
   | ChangeCustomerName(string)
+  | ChangeOrder(list(OrderItem.t))
```

Comment:
This one can be called ChangeOrderItems now that it's not actually updating the entire order anymore.